### ISSUE TITLE
Implement a streamlined method for generating lazy-loading PNG's

### ISSUE THREAD

**Issue by 356Sandhu** [GoogleWebComponents/model-viewer#928]
There should be a streamlined method for generating images that can be used for lazy loading, as it is very difficult to create an image that would look similar to the starting position of the scene. 

-- this is my first issue on github, please delete if I broke any rules.

**Comment by cdata** [comment]
@356Sandhu thanks for reporting, and welcome to our Github project. I'm glad we get to work with you on your first issue :)

We actually implemented a method specifically for this purpose in #858 / #861. We then further refined it for taking "seamless" poster images in #599 / #890. These changes will be released in v0.8.0 (which should should be released today or tomorrow).

The way to create a screenshot that allows for what we call "seamless" poster images looks like this:

```javascript
(async () => {
  const modelViewer = document.querySelector('model-viewer');
  const blob = await modelViewer.toBlob({ idealAspect: true });
  const screenshotURL = URL.createObjectURL(blob);

  // From here, you can do anything you want with the URL
  // One example would be to open it in a new tab:
  window.open(screenshotURL);

  // An alternative example would be to make a download link for it:
  const a = document.createElement('a');
  a.download = true;
  a.href = screenshotURL;
  a.textContent = 'Download screenshot';
  document.body.appendChild(a);
})();
```

**Comment by 356Sandhu** [comment]
Thank you for sharing this! I'm looking forward to using it. 😃
